### ISSUE TITLE
Merge songs (i.e. simultaneous)

### ISSUE THREAD

**Issue by jamesdeluk** [jiaaro/pydub#485]
Is it possible to merge songs, as in, have them play simultaneously (and then export)?

For example, one track is instrumental, one track is vocals, and I want to merge them into one track - not one after the other, as with '+'.

Thanks!

**Comment by jiaaro** [comment]
Yes, you're looking for [`AudioSegment(…).overlay()`](https://github.com/jiaaro/pydub/blob/master/API.markdown#audiosegmentoverlay)

**Comment by jamesdeluk** [comment]
Aha, thanks! I feel stupid for missing that. In all honesty, I'd never seen the API document - on the README, all the links under "Stuff you might be looking for:" are on the README page _except_ for API, so I was thinking that was a contents for the README page. Perhaps add another link to the API after the Quickstart section?

**Comment by rgoupil** [comment]
Since this is already in the API documentation, I reckon that issue can be closed @jiaaro?
